agent/src/beerocks/fronthaul_manager/ap_manager/ap_manager_thread.cpp

```diff
@@ -112,6 +114,7 @@ bool ap_manager_thread::create_ap_wlan_hal()
 
     bwl::hal_conf_t hal_conf;
     hal_conf.ap_acs_enabled = acs_enabled;
+    hal_conf.wpa_ctrl_path  = m_hostap_iface_ctrl;
```

Comment:
It would make a whole lot more sense to me to read the path directly from bpl_cfg here. I.e. add a function `bpl_cfg_get_hostap_ctrl_path(radio_name)` that gets the path for that radio.

However, that is a bit of an architectural change, because up to now we mostly used bpl only through the platform manager thread.